### PR TITLE
replace http links

### PR DESCRIPTION
**Pull request by mairaw**
CATS reports were listing a bunch of links that were still using http. This PR fixes some of the links we still had in our docs and fixes their format too. For constant values, I kept the http value but fixed the formatting.

Contributes to https://github.com/dotnet/docs/issues/4504

### CODE REVIEW

File: xml/System.Web.Services.Description/HttpOperationBinding.xml [hunk 1]
```diff
@@ -27,7 +27,7 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- This class respresents an extensibility element in Web Services Description Language (WSDL) specific to HTTP. It specifies the relative URL of an operation using as the base the location specified by an <xref:System.Web.Services.Description.HttpAddressBinding>. For more information about specifying protocols for XML Web services, see [&#91;\<topic://cpconbuildingaspnetwebservices>&#93;](https://msdn.microsoft.com/library/01dfc27c-c68e-4910-a0aa-5e4c2a766b0c). For more information about WSDL, see the specification at http://www.w3.org/TR/wsdl/.  
+ This class respresents an extensibility element in Web Services Description Language (WSDL) specific to HTTP. It specifies the relative URL of an operation using as the base the location specified by an <xref:System.Web.Services.Description.HttpAddressBinding>. For more information about specifying protocols for XML Web services, see [&#91;\<topic://cpconbuildingaspnetwebservices>&#93;](https://msdn.microsoft.com/library/01dfc27c-c68e-4910-a0aa-5e4c2a766b0c). For more information about WSDL, see the [WSDL](https://www.w3.org/TR/wsdl/) specification.
```
Comment (rpetrusha) [inline xml/System.Web.Services.Description/HttpOperationBinding.xml]: [&#91;\<topic://cpconbuildingaspnetwebservices>&#93;](https://msdn.microsoft.com/library/01dfc27c-c68e-4910-a0aa-5e4c2a766b0c) --> \XML Web Services Using ASP.NET\(https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-4.0/ba0z6a33(v=vs.100))

File: xml/System.Web.Services.Description/HttpUrlEncodedBinding.xml [hunk 2]
```diff
@@ -29,7 +29,7 @@
...
   
- Note that this encoding is derived from a feature of HTML and is not specific to the <xref:System.Web.UI.HtmlControls.HtmlForm> class, although the `HtmlForm` class does support it. For more information about specifying protocols for XML Web services, see [&#91;\<topic://cpconbuildingaspnetwebservices>&#93;](https://msdn.microsoft.com/library/01dfc27c-c68e-4910-a0aa-5e4c2a766b0c). For more information about Web Services Description Language (WSDL), see the specification at http://www.w3.org/TR/wsdl/.  
+ Note that this encoding is derived from a feature of HTML and is not specific to the <xref:System.Web.UI.HtmlControls.HtmlForm> class, although the `HtmlForm` class does support it. For more information about specifying protocols for XML Web services, see [&#91;\<topic://cpconbuildingaspnetwebservices>&#93;](https://msdn.microsoft.com/library/01dfc27c-c68e-4910-a0aa-5e4c2a766b0c). For more information about Web Services Description Language (WSDL), see the [WSDL](https://www.w3.org/TR/wsdl/) specification.
```
Comment (rpetrusha) [inline xml/System.Web.Services.Description/HttpUrlEncodedBinding.xml]: [&#91;\<topic://cpconbuildingaspnetwebservices>&#93;](https://msdn.microsoft.com/library/01dfc27c-c68e-4910-a0aa-5e4c2a766b0c) --> \XML Web Services Using ASP.NET\(https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-4.0/ba0z6a33(v=vs.100))

File: xml/System.Windows.Media.Imaging/BitmapMetadataBlob.xml [hunk 3]
```diff
@@ -25,10 +25,10 @@
  ]]></format>
     </remarks>
     <altmember cref="T:System.Windows.Media.Imaging.BitmapMetadata" />
-    <related type="ExternalDocumentation" href="http://www.w3.org/TR/PNG-Chunks.html">Portable Network Graphics Textual Data (tEXt) Specification</related>
+    <related type="ExternalDocumentation" href="https://www.w3.org/TR/PNG-Chunks.html">Portable Network Graphics Textual Data (tEXt) Specification</related>
     <related type="ExternalDocumentation" href="http://www.iptc.org">International Press Telecommunications Council (IPTC) Schema Information</related>
```
Comment (rpetrusha) [inline xml/System.Windows.Media.Imaging/BitmapMetadataBlob.xml]: this should be https as well.

File: xml/System.Xml.Schema/XmlSchema.xml [hunk 4]
```diff
@@ -68,8 +68,8 @@
  ]]></format>
     </remarks>
     <related type="Article" href="https://msdn.microsoft.com/library/bbbcc70c-bf9a-4f6a-af72-1bab5384a187">Working with XML Schemas</related>
```
Comment (rpetrusha) [inline xml/System.Xml.Schema/XmlSchema.xml]: if you want to fix this one: <~/docs/standard/data/xml/working-with-xml-schemas.md>

File: xml/System.Web.Services.Description/HttpOperationBinding.xml (hunk 1 above)
Comment (mairaw) [reply xml/System.Web.Services.Description/HttpOperationBinding.xml]: I've opened a separate issue to take care of this site wide: #1608

**Pull request merged**